docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Design
+
+The system is implemented as an k8s operator using the [operator-sdk](https://github.com/operator-framework/operator-sdk)
+but is deployed as a DaemonSet instead of Deployment with [filtering](https://github.com/operator-framework/operator-sdk/blob/master/doc/user/event-filtering.md) only events for the DaemonSet pod node.
+
+There are two controllers one for [Node](https://godoc.org/k8s.io/api/core/v1#Node)
```

Comment:
This is painfully outdated :/ Since you just moved this part, don't worry fixing it.

Reply:
I guess this is a part of the original file